automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -776,3 +778,20 @@ def csv_reader(component, subcommand):
         entity_list.append(row)
     comp_dict[component] = entity_list
     return comp_dict
+
+
+def create_setup_dict(dict):
+    data.update(dict)
+    print data
+    with open('{}.pkcl'.format('product_setup'), 'wb') as pref:
+        pickle.dump(data, pref)
+
+
+def get_setup_data():
+    with open('{}.pkcl'.format('product_setup')) as pref:
+        data = pickle.load(pref)
+    clients6 = data['clients6']
+    clients7 = data['clients7']
+    sat_host = data['sat_host']
+    cap_hosts = data['capsule_hosts']
+    return sat_host, cap_hosts, clients6, clients7
```

Comment:
If you are returning 4 values then at receiving end you always need 4 variables. And that is not in case of dictionary return.
Also the purpose of dictionary is being comprised here.